File: Bot/Handelers/DeathHandeler.js
const LogChannelId = require("./../config").LogChannelId

async function OnDeath(req, res, client){
    try{
        const Killer = req.body.Killer;
        const KillerId = req.body.KillerId;
        const KillerRole = req.body.KillerRole;
        const Target = req.body.Target;
        const TargetId = req.body.TargetId;
        const TargerRole = req.body.TargetRole;
        const TK = req.body.TK;
        const Weapon = req.body.Weapon

        if(TK == "kill"){
            client.channels.cache.get(LogChannelId).send(`☠️ | Hráč ${Killer} (${KillerId}) <${KillerRole}> zabil hráče ${Target} (${TargetId}) <${TargerRole}> pomocí ${Weapon}`)
        }
        else if(TK == "TK"){
            client.channels.cache.get(LogChannelId).send(`⚠️☠️ | Hráč ${Killer} (${KillerId}) <${KillerRole}> zabil spoluhráče ${Target} (${TargetId}) <${TargerRole}> pomocí ${Weapon}`)
        }

        res.sendStatus(200)
    }
    catch {
        res.sendStatus(500)
    }

}

module.exports = {OnDeath}

File: Plugin/Sitrep Remastered/Handelers/DeathHandeler.cs
using Exiled.Events.EventArgs.Player;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Exiled.API.Enums;
using System.Net.Http;
using Exiled.API.Features;
using System.IO;

namespace Sitrep_Remastered.Handelers
{
    public class DeathHandeler
    {
        public async void OnDeath(DyingEventArgs ev)
        {
            try
            {
                string Killer = "Server";
                string KillerId = "Server";
                string KillerRole = "Server";
                string TK = "kill";
    
                if (ev.Attacker != null)
                {
                    Killer = ev.Attacker.Nickname.ToString();
                    KillerId = ev.Attacker.UserId.ToString();
                    KillerRole = ev.Attacker.Role.Name.ToString();
                }

                if (ev.DamageHandler.IsFriendlyFire)
                {
                    if (ev.Attacker.Role.Team != PlayerRoles.Team.ClassD || ev.Player.Role.Team != PlayerRoles.Team.ClassD)
                    {
                        TK = "TK";
                    }
                }

                var values = new Dictionary<string, string>
                {
                    {"Killer", Killer },
                    {"KillerId", KillerId },
                    {"KillerRole", KillerRole },
                    {"Target", ev.Player.Nickname.ToString()},
                    {"TargetId", ev.Player.UserId.ToString()},
                    {"TargetRole", ev.Player.Role.Name.ToString() },
                    {"TK", TK },
                    {"Weapon", ev.DamageHandler.DeathTranslation.LogLabel}
                };

                var content = new FormUrlEncodedContent(values);

                await PluginMain.client.PostAsync("http://127.0.0.1:6868/OnDeath", content);
            }
            catch (Exception err) {
                Log.Error(err);
            }

        }
    }
}
